pkg/snowflake/generic.go

```diff
@@ -108,24 +109,53 @@ func (ab *AlterPropertiesBuilder) Statement() string {
 
 	sb.WriteString(ab.rawStatement)
 
-	for k, v := range ab.stringProperties {
-		sb.WriteString(fmt.Sprintf(" %s='%s'", strings.ToUpper(k), EscapeString(v)))
+	sortedStringProperties := make([]string, 0)
+	for k := range ab.stringProperties {
+		sortedStringProperties = append(sortedStringProperties, k)
 	}
+	sort.Strings(sortedStringProperties)
 
-	for k, v := range ab.stringListProperties {
-		sb.WriteString(fmt.Sprintf(" %s=%s", strings.ToUpper(k), formatStringList(v)))
+	for _, k := range sortedStringProperties {
```

Comment:
This is a little verbose for all the fields. Can you make a helper function that returns a sorted list of string?
```suggestion
	for _, k := range sortStrings(ab.stringProperties) {
```

`sortStrings` would look something like `func sortStrings(strs []string) []string` and would return a list of a sorted strings from the list passed in.

Reply:
@jacobrheath I copied this from the Create function which is also in Generic, but I will try your suggestion